src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -0,0 +1,311 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nit: maybe Resources.resx, since we could add other things here in the future.

Reply:
I decided to call this `String` because my intention would be to put other resource in their own resx file for organization.